src/main/java/org/traccar/broadcast/RedisBroadcastService.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2023 Anton Tananaev (anton@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.broadcast;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.traccar.config.Config;
+import org.traccar.config.Keys;
+import org.traccar.model.BaseModel;
+import org.traccar.model.Device;
+import org.traccar.model.Event;
+import org.traccar.model.Permission;
+import org.traccar.model.Position;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPubSub;
+
+public class RedisBroadcastService implements BroadcastService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RedisBroadcastService.class);
+
+    private final ObjectMapper objectMapper;
+
+    private final ExecutorService service = Executors.newSingleThreadExecutor();
+
+    private final Set<BroadcastInterface> listeners = new HashSet<>();
+
+    private final String url;
+    private final String pubsubChannel = "traccar:cast";
```

Comment:
Maybe just call it "traccar"? I would also rename the variable to just `channel`.